components/details.jsx

```diff
@@ -11,6 +12,20 @@ class Details extends React.Component {
     ReactGA.event(this.props.createGaEventConfig(`Get involved`, `Clicked`, `beacon`));
   }
 
+
+  renderTimePosted() {
+    if (!this.props.created && !this.props.publishedBy) return null;
```

Comment:
Hey @Pomax ummm spacing looks good to me with and without the "time posted" line. 

<img width="753" alt="screen shot 2017-07-21 at 11 15 16 am" src="https://user-images.githubusercontent.com/2896608/28476731-f03dce1a-6e05-11e7-9129-5df6891e753a.png">
<img width="749" alt="screen shot 2017-07-21 at 11 15 05 am" src="https://user-images.githubusercontent.com/2896608/28476736-f2e0125e-6e05-11e7-9dce-f46c3edc4436.png">